alacritty/src/config/mod.rs

```diff
@@ -232,3 +201,17 @@ fn print_deprecation_warnings(config: &Config) {
         );
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::Config;
+    use std::path::PathBuf;
+
+    #[test]
+    fn config_read_eof() {
+        let config_path: PathBuf = ["..", "alacritty.yml"].iter().collect();
```

Comment:
```suggestion
        let config_path = PathBuf::from("../alacritty.yml");
```

This should do the job, right?

Reply:
I don't think that it is going to work on Windows.

Reply:
For what it's worth, the two are equivalent as long as you're dealing with strings, so will work on windows as well due to Path and PathBuf solving the platform issues.

ref: https://doc.rust-lang.org/std/path/struct.PathBuf.html#examples

Reply:
This reference doesn't say anything about their equivalence at all. It turned out that `/` is also a separator on Windows, however this platform uses `\` as a **main** separator. `Path` and `PathBuf` doesn't solve this for me actually, it's done by Windows (or at least modern windows). However there is some edge cases, like root prefixes, `C:` and so on If I'm understanding the code in path.rs correctly.

So, the following conversion on Windows is lossless. 
```rust
let p: = std::path::PathBuf("hello/world/zoo");
let p = p.to_str();
``` 

P.s. default windows terminal perfectly understand navigation with `/` and file explorer perfectly understand navigation with them.